meta-ibm/recipes-phosphor/webui/webui-vue/change-logo.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Start the change logo script
+After=basic.targe
```

Comment:
Seeing this when booting, seems we need to be sure to run after vpd has run:

Aug 03 15:58:51 p10bmc busctl[251]: Failed to get property SubModel on interface xyz.openbmc_project.Inventory.Decorator.Asset: The name is not activatable

Reply:
Also, missing a "t" on target

Reply:
@lxwinspur We really have to make sure we test our PRs.. I don't see how this could have worked